tensorflow/python/keras/backend.py

```diff
@@ -1912,6 +1912,24 @@ def gather(reference, indices):
 
   Returns:
       A tensor of same type as `reference`.
+  
+  Examples:
+  
```

Comment:
Please make sure to remove trailing whitespace here and two lines above at least